lib/Controller/AssistantController.php

```diff
@@ -33,9 +34,10 @@ public function __construct(
 	 */
 	#[NoAdminRequired]
 	#[NoCSRFRequired]
-	#[BruteForceProtection(action: 'taskResultPage')]
+	#[BruteForceProtection(action: 'taskResults')]
```

Comment:
I continue to be flabbergasted by BruteforceProtection.. :thinking: 

[Our docs](https://docs.nextcloud.com/server/latest/developer_manual/basics/controllers.html#brute-force-protection) seem to indicate we need to pass in `['action' => 'taskResults']` to throttle?

Reply:
I think the doc is weird/wrong. What you pass to `throttle` is the additional metadata, not the action again. The action of the created entries in the bruteforce_attempts table is determined by the method attribute `action` parameter.

You can actually pass anything you want as metadata so you know more about why something was throttled.

Reply:
ah ok

Reply:
Oh I see, if there is an `action` key in the metadata array, it is used to know to which bruteforce this throttling is related (in case there are multiple protections to one method). But one can actually add whatever other keys in metadata.

In this case there is only one bruteforce protection method attr so we don't need to specify the action in the metadata.